mds/api/auth.py

```diff
@@ -1,8 +1,11 @@
 """
 Authentication module for MDS API calls.
 """
-
+from mds.api.auth import AuthorizationToken
```

Comment:
This is from this very file, so doesn't need to be imported.

Reply:
It has been corrected.